### DISCUSSION TITLE
Track connection across multiple patchpanels

### DISCUSSION THREAD

**Discussion by FisoF** [netbox-community/netbox#10661] [Q&A]
Hello , 
Its possible to fully track connectin across multiple patch panels ? 

Scheme:

![trackconnection](https://user-images.githubusercontent.com/94540299/196087587-f0254dc0-6f8d-4a04-ace5-d9b9424971b4.png)

Now result in netbox is: 

![TraceResult](https://user-images.githubusercontent.com/94540299/196088693-04307a99-5628-44b6-8d25-69b09a70c624.png)

Thaks for answer

**Comment by joaolucasmacedo** [comment]
It is totally possible to do what you showed on that picture. Looks like PatchPanel2 has not a corresponding front port for that rear port.

**Reply by FisoF** [reply]
Thank for your reply. On PatchPanel2 I have 1 rear port with 12 positions (fiber optic cable terminated with splices) Its wrong way ? I must create connection between rear ports of patch panels "line by line",  not one cable with multiple positions?  
![patchpanel2](https://user-images.githubusercontent.com/94540299/197583817-3f084c38-e4a4-4f47-8003-6ff6820fd316.png)
![patchpanel2FP](https://user-images.githubusercontent.com/94540299/197585320-84be49a7-11f2-4648-a552-248cc06354ab.png)
![patchpanel2FPdetail](https://user-images.githubusercontent.com/94540299/197585347-65efea6a-e38d-44c9-9a28-23203da8b1bb.jpg)

**Reply by joaolucasmacedo** [reply]
I think you are doing right but looks like Netbox has that limitation when you have a multi position segment on a cable trace.
See #10602 
In theory it would work as you expected, so it looks like a bug.

**Reply by CupcakeMunster** [reply]
I've done exact what you're trying to do recently and it works. Can you show what the cable between the front ports of patch panel 2 and 3 look like?

On my setup, between panel 2 and 3 I have one cable connected to both ports on both panels.

**Reply by CupcakeMunster** [reply]
> I've done exact what you're trying to do recently and it works. Can you show what the cable between the front ports of patch panel 2 and 3 look like?
> 
> On my setup, between panel 2 and 3 I have one cable connected to both ports on both panels.

![image](https://user-images.githubusercontent.com/26608204/198121088-731f62ef-ff26-4923-aba9-6d21ca771e1d.png)

**Reply by FisoF** [reply]
@CupcakeMunster my connection is same as your but...  can you show your full trace ? 
![cable34](https://user-images.githubusercontent.com/94540299/198709733-e559f3a1-1dcc-4d3f-a341-c4e1f5c1a9aa.png)

**Reply by CupcakeMunster** [reply]
I recreated it in the demo site. If you check it out before tomorrow it'll probably still be there. Look for the location called "rats nest"
![demopatching](https://user-images.githubusercontent.com/26608204/198748055-970dbfbc-30b4-4d24-ae1d-d4618ca007d7.svg)

**Comment by mtinberg** [comment]
I think in this case you should have a 1:1 relationship with front-port to rear-port, so each strand/pair of the 12-count cable terminates in a distinct rear-port that matches a single front-port.  I think the different positions are for MUXes where multiple strands of front-port fiber end up on a single strand of rear-port fiber, not for when multiple strands are bundled into a single cable.  Or am I grossly misinterpreting what you are doing here?


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________

**Reply by joaolucasmacedo** [reply]
I think that the way @FisoF is doing is right, as I understand cabling modeling in Netbox. I should work for both MUXes and multi fiber cables.
See #10602

**Comment by FisoF** [comment]
@mtinberg maybe you are right and I misunderstand cabling modeling(and different positions is realy only for MUXes/splitters https://www.peak-fiber.com/product/plc-splitter/ ) . Detailed description in documentation is missing... Or is it bug as @joaolucasmacedo says...

**Comment by mtinberg** [comment]
I'm reading through the doc and checking in the lab, I think the Positions number is used for MPO cables (and maybe also MUX) where multiple fiber strands are included in the same physical connector/port

https://demo.netbox.dev/static/docs/models/dcim/rearport/

eg https://edgeoptic.com/kb_article/mtp-mpo-cables-all-basics-you-need-to-know/

where you aren't terminating each strand individually in the fiber patch, I suppose LC works that way too.


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
